cmd/combine/combine.go

```diff
@@ -220,13 +222,19 @@ func loadManifest(ctx context.Context, dir string, noverify bool) (*manifestpb.C
 		lockFile := filepath.Join(dir, sd.Name(), "cluster-lock.json")
 		manifestFile := filepath.Join(dir, sd.Name(), "cluster-manifest.pb")
 
-		cl, _, err := manifest.Load(manifestFile, lockFile, func(lock cluster.Lock) error {
+		cl, legacyLockLoaded, err := manifest.Load(manifestFile, lockFile, func(lock cluster.Lock) error {
 			return verifyLock(ctx, lock, noverify)
 		})
 		if err != nil {
 			return nil, nil, errors.Wrap(err, "manifest load error", z.Str("name", sd.Name()))
 		}
 
+		if (didReadLegacyLock != legacyLockLoaded) && idx != 0 {
+			return nil, nil, errors.New("loaded legacy lock, but not all other directories being combined contain one")
```

Comment:
```suggestion
			return nil, nil, errors.New("missing legacy lock in some directories")
```

Reply:
Hmmm while I agree the error I proposed isn't worded the best, it highlights the fact that finding a cluster lock and no manifest is unexpected... @corverroos you're the error message magician, do you have any suggestion?